### DISCUSSION TITLE
Preconnect origins in response header link

### DISCUSSION THREAD

**Discussion by Monse132** [GoogleChrome/lighthouse#13197] [Q&A]
Hi
I have been generating tests in page speed insights and lighthouse of the URL https://www.newhomesource.com/plan/kenlie-betenbough-homes-amarillo-tx/1886218 mobile.
In Passed audits section I get a warning related to preconnect 
![image](https://user-images.githubusercontent.com/87142295/136622111-7f5637db-a54b-4da6-b6f7-959bf11cbf50.png)

We are  preconnecting several domains in server side in the response header link.
![image](https://user-images.githubusercontent.com/87142295/136622447-cd749840-d7ad-4b8c-980a-b58dcf2a5506.png)

When I remove that preconnect in server side , the audit don´t warn me.

Let me know if it is a lighthouse bug because the origins preconnect are criticial for our site.

Regards

**Comment by FernandoSolano20** [comment]
Same happens to me
